File: cdk-from-cfn-testing/expected/sam_nodejs_lambda_arr_transform/typescript/SAMNodeJSLambdaArrConstruct.ts
import * as cdk from 'aws-cdk-lib';
import * as sam from 'aws-cdk-lib/aws-sam';
import { Construct } from 'constructs';

export interface SAMNodeJSLambdaArrConstructProps {
}

export class SAMNodeJSLambdaArrConstruct extends Construct {
  public constructor(scope: Construct, id: string, props: SAMNodeJSLambdaArrConstructProps = {}) {
    super(scope, id);

    // Transforms
    cdk.Stack.of(this).addTransform('AWS::Serverless-2016-10-31');

    // Resources
    const myFunction = new sam.CfnFunction(this, 'MyFunction', {
      runtime: 'nodejs20.x',
      handler: 'index.handler',
      inlineCode: 'exports.handler = async (event) => {\n  console.log(event);\n}\n',
    });
  }
}

File: cdk-from-cfn-testing/expected/sam_nodejs_lambda_arr_transform/python/SAMNodeJSLambdaArrConstruct.py
from aws_cdk import Stack
import aws_cdk as cdk
import aws_cdk.aws_sam as sam
from constructs import Construct

class SAMNodeJSLambdaArrConstruct(Construct):
  def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
    super().__init__(scope, construct_id)

    # Transforms
    Stack.of(self).add_transform('AWS::Serverless-2016-10-31')

    # Resources
    myFunction = sam.CfnFunction(self, 'MyFunction',
          runtime = 'nodejs20.x',
          handler = 'index.handler',
          inline_code = 'exports.handler = async (event) => {\n  console.log(event);\n}\n',
        )


